src/services/auth.service.js

```diff
@@ -5,27 +5,40 @@ const mongoose = require('mongoose')
 mongoose.Promise = Promise
 
 const User = require('../models/user.model')
-const tokenLife = config.TOKEN_LIFE
+const tokenExpTime = config.TOKEN_LIFE
+const moment = require('moment')
 const _ = require('lodash')
 
 const randomBytes = Promise.promisify(require('crypto').randomBytes)
 
 module.exports = () => {
   const generateToken = user =>
     jwt.sign(user, config.SECRET, {
-      expiresIn: parseInt(tokenLife)
+      expiresIn: parseInt(tokenExpTime)
     })
 
   const getToken = user => `JWT ${generateToken(user)}`
 
+  const setExpirationDate = () =>
+    moment().add((parseInt(tokenExpTime) / 3600), 'h')
```

Comment:
We're setting `tokenExpTime` in seconds. That's the formula to convert seconds to hours. This function adds `tokenExpTime` (in hours) to the current date.